tests/test_api_frame.py

```diff
@@ -5,12 +5,12 @@
 
 import jaxsim.api as js
 from jaxsim import VelRepr
+from jaxsim.math.quaternion import Quaternion
 
 from . import utils_idyntree
 
 
 def test_frame_index(jaxsim_models_types: js.model.JaxSimModel):
-
```

Comment:
Nit: can you keep spaces?

Reply:
Thanks, solved in https://github.com/ami-iit/jaxsim/pull/210/commits/beed30b3a584a538fe5d97f9dad30238defda4ff